app/javascript/application/images/copy-link.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@BALAJI-BALA - inside the images folder, perhaps we can create another folder for `icons`? So it's organised better